app/views/carriers/_form.haml

```diff
@@ -0,0 +1,6 @@
+.row
+  .col-md-6.offset-md-3
+    = bootstrap_form_with model: @carrier, local: true do |f|
+      = f.text_field :name
+      = f.text_field :gateway_address
```

Comment:
If I were filling this out, I'd wonder if I should put the `@` in this text field or if it'd be filled in. Later on, your code assumes that they do begin with the `@`. Could you validate that the gateway address begins with `@`?